INSTALL.md

```diff
@@ -40,35 +40,29 @@ Dangerzone is available for:
 
 Add our repository following these instructions:
 
-Install `curl`, if it's not already available:
+Install `gpg`, if it's not already available:
 
 ```sh
-sudo apt-get install -y curl
+sudo apt-get install -y gpg ca-certificates
 ```
 
 Download the GPG key for the repo:
 
 ```sh
+gpg --keyserver hkps://keys.openpgp.org \
+    --no-default-keyring --keyring ./fpf-apt-tools-archive-keyring.gpg \
```

Comment:
Should we save it in `/tmp/fpf-apt-tools-archive-keyring.gpg` to avoid messing up the user's `$PWD` directory?

Reply:
It's generally best to avoid saving sensitive files in a world-writeable directory with a predictable name. Chances are slim, but another low-privileged process can add its own keyring there.

Other ideas I had in mind:

1. Use `sudo gpg` and store the keyring directly under `/etc/apt/keyrings/fpf-apt-tools-archive-keyring.gpg`. This is short and sweet but also dangerous. I wouldn't trust running `gpg` with `sudo` due to any bugs lurking there.
2. Generate a random filename, save it in an environment variable, and use that name in the `gpg` and `mv` commands. This makes things a bit more complicated to users reading these instructions.
3. Let `gpg` store the keyring under `~/.gnupg/`, by not specifying a path (i.e., remove `./`). See also the [documentation on `--keyring`](https://www.gnupg.org/gph/en/manual/r1574.html). I'm not sure though what are the repercussions of storing a keyring under `~/.gnupg`. I can say that the default keyring is not affected, so that's a good indication.

All in all, I think chances are slim that the user's `$PWD` will be affected, since we immediately move the file to the target path. If we ever have complaints though, we can revisit this decision.

Reply:
Ah. Right. We're moving it anyways. Let's keep it then.